### ISSUE TITLE
Add a quick guide for setting up the remote (and LXC) spawners

### ISSUE THREAD

**Issue by pevogam** [avocado-framework/avocado#5838]
Let's reuse @wsakernel's effort in documenting some crucial steps for a new user of the remote (and LXC) spawners. Instructions are as follows:

First add the slot to your avocado config file:
```
    [spawner.remote]
    slots = ['board']
```
Then you need a JSON file of the same name as the slot. Its contents are the command line parameters of the ``remote_login`` function of class ``remote``, e.g.:
```
    {
        "client": "telnet",
        "host": "192.168.64.2",
        "port": "23",
        "username": "root",
        "password": "",
        "prompt": "#"
    }
```
Final important detail: the remote site also needs avocado installed.

**Comment by pevogam** [comment]
I am mostly leaving this for the future where we improve some of the new user experience before @wsakernel's output gets lost deep in the comments of a merged pull request.

**Comment by clebergnu** [comment]
> I am mostly leaving this for the future where we improve some of the new user experience before @wsakernel's output gets lost deep in the comments of a merged pull request.

Thanks for this @pevogam. I've added your text as close to verbatim as possible to #5846

**Comment by pevogam** [comment]
I actually only meant to keep track of this for now for post-LTS since I did not want to add any additional issues to the respective milestone but thank a lot for adding it!

**Comment by richtja** [comment]
solved in  #5846
